<?xml version="1.0"?>

<launch>
<!-- Lancia i driver del sonar-->
<group ns="srf_serial">
  <node pkg="srf_serial" type="srf_serial_node" name="sensor" output="screen" 
  > <!--launch-prefix="gdbserver localhost:10000" -->
	<param name="serial_device" type="string" value="/dev/srf08"/>
	<param name="update_frequency" value="21"/>
	<param name="sensor_type" type="string" value="srf08"/>
    <param name="i2c_address" type="string" value="0xe0"/>
   	<!-- Uncomment and start node once in order to change the address,
    do not forget to change to configuration afterwards -->
    <!-- <param name="change_to_i2c_address" type="string" value="0xe2"/> -->
    <param name="amplification" value="17"/>
    <param name="enable_median_filter" value="true"/>
    <param name="low_pass_smoothing" value="0.8"/>
  </node>
</group>

<!--Nodo TF-->
<node pkg="lidar_cam_mapping" type="altimeter_tf" name="altimeter_tf" output="screen">
	<remap from="range_topic" to="/srf_serial/sensor/sonar" />
	<param name="base_stabilized_frame" type="string" value="base_stabilized" />
	<param name="base_footprint_frame" type="string" value="base_footprint" />
</node>

</launch>
